tests/stdlib/tordsets.nim

```diff
@@ -0,0 +1,97 @@
+import ordsets
```

Comment:
new modules should not be importable without `std/` prefix to avoid name clashes with future nimble packages; maybe ordsets should be under `std/ordsets` even though it's a collection

Reply:
What does this entail? moving the ordsets.nim to Nim/lib/std/ folder?

Reply:
it's marked as resolved but was it resolved? ;-)

Reply:
nevermind, i hadn't clicked refresh!